support/Pipelines/Bbh/InitialData.py

```diff
@@ -200,6 +205,79 @@ def generate_id(
     )
 
 
+def postprocess_id(
+    id_input_file_path: Union[str, Path],
+    id_run_dir: Optional[Union[str, Path]] = None,
+    evolve: bool = False,
+    pipeline_dir: Optional[Union[str, Path]] = None,
+    **scheduler_kwargs,
+):
+    """Postprocess initial data after generation.
+
+    This function is called after the initial data has been generated (see the
+    'Next' section in the 'InitialData.yaml' input file). It does the following:
+
+    - Find apparent horizons in the data to determine quantities like the masses
+      and spins of the black holes. These quantities are stored in
+      'BbhReductions.h5' in subfiles 'Ah{A,B}.dat'. In addition, the horizon
+      surface coordinates and coefficients are written to 'BbhSurfaces.h5' in
+      subfiles 'Ah{A,B}/Coordinates' and 'Ah{A,B}/Coefficients'.
+
+    Then, the inspiral evolution is started if 'evolve' is set to True.
+    """
+    from spectre.Pipelines.Bbh.FindHorizon import find_horizon
+
+    # Read input file
+    with open(id_input_file_path, "r") as open_input_file:
+        _, id_input_file = yaml.safe_load_all(open_input_file)
+    x_B, x_A = id_input_file["Background"]["Binary"]["XCoords"]
+    id_domain = id_input_file["DomainCreator"]["BinaryCompactObject"]
+    excision_radius_A = id_domain[f"ObjectA"]["InnerRadius"]
+    excision_radius_B = id_domain[f"ObjectB"]["InnerRadius"]
+
+    # Find latest observation in output data
+    if id_run_dir is None:
+        id_run_dir = Path(id_input_file_path).resolve().parent
+    id_volfiles = glob.glob(str(Path(id_run_dir) / "BbhVolume*.h5"))
+    obs_id, _ = select_observation(
+        open_volfiles(id_volfiles, "VolumeData"), step=-1
+    )
+
+    # Find horizons and store the data in the output files
+    for object_label, xcoord, excision_radius in zip(
+        ["AhA", "AhB"], [x_A, x_B], [excision_radius_A, excision_radius_B]
+    ):
+        find_horizon(
+            id_volfiles,
+            subfile_name="VolumeData",
+            obs_id=obs_id,
+            obs_time=0.0,
+            initial_guess=Strahlkorper(
+                l_max=12,
```

Comment:
I don't think we should expose the m_max option. Everything goes unstable if m_max < l_max. Probably the right thing to do is to just not expose the constructor from C++.

Reply:
Yeah probably not, but you're right, this is more an issue with the c++ binding. I think we can do this in a follow-up PR.

We should probably make LMax an option (and default it to something like 12). This won't matter as much once the horizon finder is adaptive, but for now would be nice to have the flexibility.

Reply:
I switched the binding to the constructor without `m_max` and added an `l_max` option.